defaults/main.yml

```diff
@@ -36,6 +36,12 @@ stackdriver_apache_servers:
 stackdriver_elasticsearch_enabled: no
 stackdriver_elasticsearch_url: http://localhost:9200/_nodes/_local/stats/
 
+### jvm
+
+stackdriver_jvm_enabled: no
+stackdriver_jvm_host: localhost
+stackdriver_jvm_port: 5000
```

Comment:
the JVM port would be defined by each individual user based on their JVM deployment. This should definitely be parameterized and not hard coded. It would also be useful to have a check to ensure that jvm monitoring isn't activated if a port is not explicitly defined (even if the 'stackdriver_jvm_enabled' is set to 'yes'.